scripts/usb_reset.py

```diff
@@ -271,7 +271,7 @@ def mount(source, target, fs, flags=0):
 
 def umount(target):
     if ctypes.CDLL(ctypes.util.find_library("c"), use_errno=True
-                   ).umount(target) < 0:
```

Comment:
We might consider encapsulating these two functions as follows. 
The Mounter class encapsulates the loading of libc and the management of mount/umount methods, that could enhance the code's reusability and won't need to loading libc repeatly.
```
class Mounter:
    def __init__(self):
        self.libc = ctypes.CDLL(ctypes.util.find_library("c"), use_errno=True)

    def mount(self, source, target, fs, flags=0):
        if self.libc.mount(source.encode(), target.encode(), fs.encode(), flags, None) < 0:
            error_message = "Failed to mount {} ({}) to {} with flags {}: {}".format(
                source, fs, target, flags, os.strerror(ctypes.get_errno())
            )
            logging.error(error_message)
            return error_message
        else:
            return "Mount successful"

    def umount(self, target, flags=0):
        if self.libc.umount(target.encode(), flags) < 0:
            error_message = "Failed to unmount {} with flags {}: {}".format(
                target, flags, os.strerror(ctypes.get_errno())
            )
            logging.error(error_message)
            return error_message
        else:
            return "Unmount successful"
```

BTW, as libc manual mentioned https://www.gnu.org/software/libc/manual/html_node/Mount_002dUnmount_002dRemount.html
```
umount does the same thing as umount2 with flags set to zeroes. It is more widely available than umount2 but since it lacks the possibility to forcefully unmount a filesystem is deprecated when umount2 is also available.
```
Should we consider using `umount2` instead?